### PR TITLE
Static analysis

### PR DESCRIPTION
**Pull request by avandeursen**
Various static analysis improvements, possible after dropping Cobertura thus allowing use of Java 8 features such as lambdas.

### CODE REVIEW

**Comment by casperboone** [comment]
Good improvements, was just working on the same things https://github.com/SERG-Delft/jpacman-framework/commit/4beab12c3934ac5cbc49a15751185ea37c7cfc42 😛 .

File: src/main/java/nl/tudelft/jpacman/Launcher.java [hunk 1]
```diff
@@ -168,14 +168,10 @@ protected void addSinglePlayerKeys(final PacManUiBuilder builder) {
...
 	private Action moveTowardsDirection(Direction direction) {
-		return new Action() {
-
-			@Override
-			public void doAction() {
-				assert game != null;
-				getGame().move(getSinglePlayer(getGame()), direction);
-			}
-		};
+		return () -> {
```
Comment (casperboone) [inline src/main/java/nl/tudelft/jpacman/Launcher.java]: Indentation looks a bit off here, maybe change it to
```java
private Action moveTowardsDirection(Direction direction) {
    return () -> {
        assert game != null;
        getGame().move(getSinglePlayer(getGame()), direction);
    };
}
```

**Comment by avandeursen** [comment]
Ah... we should have synced.

File: src/main/java/nl/tudelft/jpacman/Launcher.java (hunk 1 above)
Comment (avandeursen) [inline src/main/java/nl/tudelft/jpacman/Launcher.java]: Yes, the space/tab thing is a bit of a mess atm....

**Pull request merged**